packages/datakit/datakit.0.10.1/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "1.2"
+maintainer:   "thomas@gazagnaire.org"
+authors:      ["Thomas Leonard" "Magnus Skjegstad"
+               "David Scott" "Thomas Gazagnaire"]
+license:      "Apache"
+homepage:     "https://github.com/moby/datakit"
+bug-reports:  "https://github.com/moby/datakit/issues"
+dev-repo:     "https://github.com/moby/datakit.git"
+doc:          "https://docker.github.io/datakit/"
+
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+build-test: ["jbuilder" "runtest" "tests/%{name}%"]
+
+depends: [
+  "jbuilder" {build}
+  "cmdliner"
+  "rresult" "astring" "fmt" "asetmap"
+  "git"         {>= "1.9.3"}
+  "uri"
+  "irmin"       {>= "1.1.0"}
+  "irmin-git"   {>= "1.0.0"}
+  "camlzip"     {>= "1.06"}
+  "cstruct"     {>= "2.2"}
+  "result"
+  "lwt"         {>= "3.0.0"}
+  "conduit" "mirage-flow"
+  "named-pipe"  {>= "0.4.0"}
+  "hvsock"      {>= "0.8.1"}
+  "logs"        {>= "0.5.0"}
+  "win-eventlog"
+  "asl"         {>= "0.10"}
+  "mtime"
+  "irmin-watcher" {>= "0.2.0"}
+  "prometheus-app"
+  "datakit-server" {>= "0.10.0"}
```

Comment:
oh because they would conflict due to protocol-9p.  seems to work by accident, but it does work :P